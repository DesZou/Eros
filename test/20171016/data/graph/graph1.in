5 4 2
4 2 825
3 1 23
4 5 570
3 2 279
10
3 4
2 5
5 1
3 5
5 3
1 2
1 4
3 4
5 4
4 2
