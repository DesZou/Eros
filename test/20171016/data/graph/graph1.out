1104
1395
-1
-1
-1
302
-1
1104
-1
-1
